lifecycle/lifecycle/deployer/redeploy.py

```diff
@@ -22,6 +22,7 @@ def redeploy_job(
     plugin_engine: PluginEngine,
     deployer_username: str,
     auth_subject: Optional[models.AuthSubject],
+    build_flags: list[str] = [], # Setting a default here let's us ignore build_flags when redeploying
```

Comment:
Is there a case where we'd ever want build flags when redeploying?

Reply:
Not for now, no.